apps/notifications/src/standalone/index.js

```diff
@@ -78,14 +78,12 @@ const customMiddleware = {
 const render = () => {
 	ReactDOM.render(
 		React.createElement( AuthWrapper( Notifications ), {
-			clientId: 56641,
```

Comment:
These props are only ever used when we need the OAuth client, so I thought it best to move them into that specific client factory instead of passing them in.

Reply:
Yes, that's what I planned to suggest after sleeping on it 🙂 `clientId` and `redirectPath` can be hardcoded into the OAuth factory function. They never change anyway.